packages/express/mixin.core.js

```diff
@@ -8,37 +8,20 @@ const { Mixin } = require('@untool/core');
 const uri = require('./lib/uri');
 
 class ExpressMixin extends Mixin {
-  create(method) {
-    const create = require(method === 'static'
-      ? './lib/static'
-      : './lib/serve');
+  createServer(mode) {
+    const create = require(mode === 'static' ? './lib/static' : './lib/serve');
     const { options, config, initializeServer, finalizeServer } = this;
-    return create(method, options, config, initializeServer, finalizeServer);
+    return create(mode, options, config, initializeServer, finalizeServer);
   }
-  run(method) {
+  runServer(mode) {
     const run = require('./lib/run');
     const { config, inspectServer, logInfo, logError } = this;
-    const app = this.create(method);
+    const app = this.createServer(mode);
```

Comment:
Why not pass a fully configured `express app` as arguments to the `runServer()` method?

Reply:
Certainly possible, but in my opinion, that would somewhat deteriorate our API: one would have to call two methods instead of just one..:

```javascript
const { createServer, runServer } = this;
runServer(createServer(mode));
```

...versus...

```javascript
this.runServer(mode);
```

I personally prefer the latter, but I will make sure you can also use the former: all it takes is a `typeof` check.

Reply:
Ah, all right. :)